hikari/impl/interaction_server.py

```diff
@@ -123,6 +138,27 @@ def status_code(self) -> int:
 )
 
 
+class _Payload(aiohttp.Payload):
```

Comment:
```suggestion
class _FilePayload(aiohttp.Payload):
```
Just to be a bit more descriptive